<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChangerWindow</class>
 <widget class="QDialog" name="ChangerWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="pin5">
   <property name="geometry">
    <rect>
     <x>930</x>
     <y>210</y>
     <width>100</width>
     <height>180</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="ninepins.qrc">
     <normaloff>:/images/kolok1_big.png</normaloff>
     <normalon>:/images/kolok2_big.png</normalon>:/images/kolok1_big.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>50</width>
     <height>110</height>
    </size>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pin4">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>210</y>
     <width>100</width>
     <height>180</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="ninepins.qrc">
     <normaloff>:/images/kolok1_big.png</normaloff>
     <normalon>:/images/kolok2_big.png</normalon>:/images/kolok1_big.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>50</width>
     <height>110</height>
    </size>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLCDNumber" name="scoreLCD">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>450</y>
     <width>200</width>
     <height>131</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pin6">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>90</y>
     <width>100</width>
     <height>180</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="ninepins.qrc">
     <normaloff>:/images/kolok1_big.png</normaloff>
     <normalon>:/images/kolok2_big.png</normalon>:/images/kolok1_big.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>50</width>
     <height>110</height>
    </size>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pin1">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>290</y>
     <width>100</width>
     <height>180</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="ninepins.qrc">
     <normaloff>:/images/kolok1_big.png</normaloff>
     <normalon>:/images/kolok2_big.png</normalon>:/images/kolok1_big.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>50</width>
     <height>110</height>
    </size>
   </property>
   <property name="shortcut">
    <string notr="true"/>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="autoRepeat">
    <bool>false</bool>
   </property>
   <property name="autoExclusive">
    <bool>false</bool>
   </property>
   <property name="autoDefault">
    <bool>true</bool>
   </property>
   <property name="default">
    <bool>false</bool>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLCDNumber" name="roundsLCD">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>450</y>
     <width>200</width>
     <height>131</height>
    </rect>
   </property>
  </widget>
  <widget class="QLCDNumber" name="pointsLCD">
   <property name="geometry">
    <rect>
     <x>800</x>
     <y>450</y>
     <width>200</width>
     <height>131</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pin3">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>210</y>
     <width>100</width>
     <height>150</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="ninepins.qrc">
     <normaloff>:/images/kolok1_big.png</normaloff>
     <normalon>:/images/kolok2_big.png</normalon>:/images/kolok1_big.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>50</width>
     <height>110</height>
    </size>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>410</y>
     <width>211</width>
     <height>37</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Verdana</family>
     <pointsize>22</pointsize>
    </font>
   </property>
   <property name="acceptDrops">
    <bool>false</bool>
   </property>
   <property name="layoutDirection">
    <enum>Qt::RightToLeft</enum>
   </property>
   <property name="text">
    <string>Súčet  kolkov</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pin8">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>-30</y>
     <width>100</width>
     <height>180</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="ninepins.qrc">
     <normaloff>:/images/kolok1_big.png</normaloff>
     <normalon>:/images/kolok2_big.png</normalon>:/images/kolok1_big.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>50</width>
     <height>110</height>
    </size>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>410</y>
     <width>201</width>
     <height>37</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Verdana</family>
     <pointsize>22</pointsize>
    </font>
   </property>
   <property name="acceptDrops">
    <bool>false</bool>
   </property>
   <property name="layoutDirection">
    <enum>Qt::RightToLeft</enum>
   </property>
   <property name="text">
    <string>Počet hodov</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pin2">
   <property name="geometry">
    <rect>
     <x>800</x>
     <y>290</y>
     <width>100</width>
     <height>180</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="ninepins.qrc">
     <normaloff>:/images/kolok1_big.png</normaloff>
     <normalon>:/images/kolok2_big.png</normalon>:/images/kolok1_big.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>50</width>
     <height>110</height>
    </size>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pin0">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>450</y>
     <width>100</width>
     <height>180</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="ninepins.qrc">
     <normaloff>:/images/kolok1_big.png</normaloff>
     <normalon>:/images/kolok2_big.png</normalon>:/images/kolok1_big.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>50</width>
     <height>110</height>
    </size>
   </property>
   <property name="autoDefault">
    <bool>true</bool>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pin7">
   <property name="geometry">
    <rect>
     <x>800</x>
     <y>90</y>
     <width>100</width>
     <height>180</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="ninepins.qrc">
     <normaloff>:/images/kolok1_big.png</normaloff>
     <normalon>:/images/kolok2_big.png</normalon>:/images/kolok1_big.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>50</width>
     <height>110</height>
    </size>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="scorePlus">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>210</y>
     <width>90</width>
     <height>90</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>30</pointsize>
    </font>
   </property>
   <property name="text">
    <string>+</string>
   </property>
  </widget>
  <widget class="QPushButton" name="scoreMinus">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>320</y>
     <width>90</width>
     <height>90</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>30</pointsize>
    </font>
   </property>
   <property name="text">
    <string>-</string>
   </property>
  </widget>
  <widget class="QPushButton" name="roundsPlus">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>210</y>
     <width>90</width>
     <height>90</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>30</pointsize>
    </font>
   </property>
   <property name="text">
    <string>+</string>
   </property>
  </widget>
  <widget class="QPushButton" name="roundsMinus">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>320</y>
     <width>90</width>
     <height>90</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>30</pointsize>
    </font>
   </property>
   <property name="text">
    <string>-</string>
   </property>
  </widget>
  <widget class="QPushButton" name="okButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>220</width>
     <height>80</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>22</pointsize>
    </font>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancelButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>110</y>
     <width>220</width>
     <height>80</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>22</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Zrušiť</string>
   </property>
   <property name="default">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="zeroButton">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>10</y>
     <width>211</width>
     <height>81</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>22</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Vynuluj</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="ninepins.qrc"/>
 </resources>
 <connections/>
</ui>
